frmts/gtiff/geotiff.cpp

```diff
@@ -1132,6 +1132,84 @@ CPLString GTiffGetCompressValues(bool &bHasLZW, bool &bHasDEFLATE,
     return osCompressValues;
 }
 
+/************************************************************************/
+/*                    OGRGTiffDriverGetSubdatasetInfo()                 */
+/************************************************************************/
+
+struct OGRGTiffDriverSubdatasetInfo : public GDALSubdatasetInfo
```

Comment:
should rather be:
```suggestion
struct GDALGTiffDriverSubdatasetInfo : public GDALSubdatasetInfo
```
or maybe just:
```suggestion
struct GTiffDriverSubdatasetInfo : public GDALSubdatasetInfo
```